src/timepicker/timepicker.stories.ts

```diff
@@ -45,6 +45,33 @@ storiesOf("Time Picker", module)
 			theme: select("Theme", ["dark", "light"], "dark")
 		}
 	}))
+	.add("With ngModel", () => ({
+		template: `
+		<ibm-timepicker [theme]="theme"
+			(valueChange)="timePickerChange($event)"
+			[(ngModel)]="model"
+			[disabled]="disableTime"
+			label="Select a time">
+			<ibm-timepicker-select [theme]="theme" (valueChange)="timePickerSelectChange($event)" [disabled]="disabledSelect" display="inline">
```

Comment:
for completeness we should also demo models attached to both `timepicker-select`s